.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": ["es2015", "stage-2"],
```

Comment:
>> Why'd you choose stage 2?

It was just and arbitrary starting point. We can adjust if we find language features we want to use but find missing. The process is quite interesting, as it appears features are moving in / out of stages based on the TC39 process. https://github.com/tc39/proposals.